### DISCUSSION TITLE
Creating CRT effects the hard way with FFmpeg CRT Transform and pairing it with NTSC-RS.

### DISCUSSION THREAD

**Discussion by TheSystemGuy1337** [ntsc-rs/ntsc-rs#198] [Show and tell]
This is my very first post here and it's going to be a bit rough, but here goes. I had been using FFmpeg CRT Transform (ffcrt) for a while now to create CRT effects without using shaders. Then I discovered NTSC-RS. So I paired them together. Using the paltv config and the Worn-Tape and Low Power NTSC preset, I was able to generate this.

![SECONDTONONE_ntsc_worntape_paltv](https://github.com/user-attachments/assets/7c3f9e4e-8531-4df3-8bdc-437fd977b550)

Another example, using the same paltv config for ffcrt, but only Worn-Tape for NTSC-RS
![Chevrolet Corvette Never Before_ntscworntape_paltv](https://github.com/user-attachments/assets/6fc7f1b0-c9ce-4fc1-be51-7edb3b9b66da)

**Comment by Degamisu** [comment]
Aamazing result! very realistic.

**Reply by TheSystemGuy1337** [reply]
If you want to give it a try, I'll leave the GitHub repo here. Keep in mind, it hasn't been updated in 4 years and is **WINDOWS ONLY.**

https://github.com/viler-int10h/FFmpeg-CRT-transform

**Comment by smfthe1199** [comment]
That's great
